csa/csa_test.go

```diff
@@ -0,0 +1,47 @@
+package csa
+
+import (
+	"testing"
+)
+
+func select_(csa Csa, k int, c int) int {
```

Comment:
???

Reply:
for bitmaps: https://pkg.go.dev/github.com/RoaringBitmap/roaring?utm_source=godoc#Bitmap.Select

Reply:
resolved